server.go

```diff
@@ -1812,14 +1812,49 @@ func (s *Server) WriteSeries(database, retentionPolicy string, points []Point) (
 				codecs[measurement.Name] = codec
 			}
 
+			// Read raw encoded series data.
+			data, err := sh.readSeries(series.ID, p.Timestamp.UnixNano())
```

Comment:
Unfortunately readSeries is a bug method only. We don't use it in production code.